RebuyFormsRating/Models/RatingViewHandler.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using RebuyFormsRating.Common.Services;
+using Xamarin.Forms;
+using Acr.UserDialogs;
+using System.Threading.Tasks;
+
+namespace RebuyFormsRating.Models
+{
+    public class RatingViewHandler
+    {
+        public bool IsRated {
+            get { return Settings.IsRated; }
+            set { Settings.IsRated = value; }
+        }
+
+        public int UsesBeforeRating { set; get; } = 1;
+
+        public int UsageCount {
+            get { return Settings.UsageCount; }
+            set { Settings.UsageCount = value; }
+        }
+
+        public string VersionNumber {
+            get { return Settings.VersionNumber; }
+            set { Settings.VersionNumber = value; }
+        }
+
+        public void CheckOpenRatingView(string appStoreId)
+        {
+            if (String.IsNullOrWhiteSpace(VersionNumber) || !VersionNumber.Equals(DependencyService.Get<IInfoService>().AppVersionCode)) {
```

Comment:
Second if-expression could be written more clearly as:

`VersionNumber != DependencyService.Get<IInfoService>().AppVersionCode`.

I'm not sure the first if-expression is necessary too, because can the AppVersionCode ever be null or whitespace?